integration/env/common.go

```diff
@@ -64,45 +61,40 @@ func init() {
 		panic(fmt.Sprintf("env var %q must not be empty", EnvVarGithubBotToken))
 	}
 
-	testedVersion = os.Getenv(EnvVarTestedVersion)
-	if testedVersion == "" {
-		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarTestedVersion))
-	}
-
 	registryPullSecret = os.Getenv(EnvVarRegistryPullSecret)
 	if registryPullSecret == "" {
 		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarRegistryPullSecret))
 	}
 
 	testDir = os.Getenv(EnvVarTestDir)
 
-	params := &framework.VBVParams{
-		Component: component,
-		Provider:  provider,
-		Token:     githubToken,
-		VType:     TestedVersion(),
-	}
-	versionBundleVersion, err = framework.GetVersionBundleVersion(params)
-	if err != nil {
-		panic(err.Error())
-	}
-	// TODO there should be a not found error returned by the framework in such
-	// cases.
-	if VersionBundleVersion() == "" {
-		if strings.ToLower(TestedVersion()) == "wip" {
-			log.Println("WIP version bundle version not present, exiting.")
-			os.Exit(0)
+	{
+		vbs := service.NewVersionBundles()
+
+		if path.Base(testDir) == "update" {
```

Comment:
I don't necessary like this part. Is `update` test enough reason to keep `TESTED_VERSION` env var? I think this should be sorted inside the test. Also this won't be like that in flattened versions because we won't have version bundles there. I wouldn't be keen to invest more time into it because of that.

Reply:
+1 on having the logic in the test. But I think it can be done as a later PR and also not worth investing lots of time here.

Reply:
Let's sit together and think about a solution for the update tests.